ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/private/migration-from-public/provision/cucushift-installer-rehearse-aws-ipi-private-migration-from-public-provision-chain.yaml

```diff
@@ -0,0 +1,30 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-private-migration-from-public-provision
+  steps:
+  - ref: ipi-conf
+  - ref: ipi-conf-telemetry
+  - ref: aws-provision-vpc-shared
+  - chain: aws-provision-bastionhost
+  - ref: ipi-conf-aws-custom-vpc
+  - ref: ipi-conf-aws
+  - ref: ipi-install-monitoringpvc
+  - ref: ipi-conf-aws-usage-info
+  - chain: ipi-install
```

Comment:
As per https://github.com/openshift/release/pull/58859, `- chain: aws-provision-iam-user-minimal-permission` is added to each AWS provision chain, before `ipi-install`
But I think `aws-provision-public-to-private` should use common creds, as no permission requirement is mentioned in the doc.